docs/developing.rst

```diff
@@ -0,0 +1,58 @@
+Developing
+==========
+
+Install the requirements
+
+::
+
+    $ pip install -r requirements.txt
+
+(note that lxml, used in the tests, requires that libxslt and libxml2 be
```

Comment:
Instead of parentheses, we could have this in a Note box similar to what is seen on http://django-name.readthedocs.org/en/migrations/installation.html. We also need to mention the requirement of Python development package